packages/cycle-scripts-one-fits-all/template/src/pages/page1.tsx

```diff
@@ -0,0 +1,75 @@
+import xs, { Stream } from 'xstream';
+import { VNode, DOMSource } from '@cycle/dom';
+import { StateSource } from 'cycle-onionify';
+
+import { DriverSources, DriverSinks } from '../drivers';
+
+export interface State {
+    count : number;
+}
+export type Reducer = (prev  State) => State | undefined;
+export type Sources = DriverSources & { onion : StateSource<State> };
+export type Sinks = DriverSinks & { onion : Stream<Reducer> };
+
+export function Page1(sources : Sources): Sinks {
```

Comment:
I was being laxzy and originally had 2 identical pages. Ispent so long trying to build and test locally tha tI ran out out steam. But you are quite right, better they are different. Thanks for the prod :)

I decidedto call them pages and put in own folder, even though they *are* components. If only as their use is a little different and so use may be specific. I also had a components folder but as was empty git ignored it . Do you think that is a worthless distinction then?